p/themes/Mapco/mapco.css

```diff
@@ -541,7 +541,7 @@ form th {
 }
 .tree .tree-folder .tree-folder-title .title {
 	background: inherit;
-	color: #fff;
+	color: #ffffff;
```

Comment:
This is all on purpose btw? I kind of like the shorthand. :-) (But either way.)

Reply:
Indeed the shorthand would be better, but there is code such as the following, which would break:

https://github.com/FreshRSS/FreshRSS/blob/9012db01554b9ac3b99651be8a3944668757fa64/p/themes/Ansum/_components.scss#L146

Reply:
Makes sense.